lib/elixir/lib/list.ex

```diff
@@ -591,15 +591,20 @@ defmodule List do
   """
   @spec to_string(:unicode.charlist) :: String.t
   def to_string(list) when is_list(list) do
-    case :unicode.characters_to_binary(list) do
-      result when is_binary(result) ->
-        result
-
-      {:error, encoded, rest} ->
-        raise UnicodeConversionError, encoded: encoded, rest: rest, kind: :invalid
-
-      {:incomplete, encoded, rest} ->
-        raise UnicodeConversionError, encoded: encoded, rest: rest, kind: :incomplete
+    try do
+      case :unicode.characters_to_binary(list) do
+        result when is_binary(result) ->
+          result
+
+        {:error, encoded, rest} ->
+          raise UnicodeConversionError, encoded: encoded, rest: rest, kind: :invalid
+
+        {:incomplete, encoded, rest} ->
+          raise UnicodeConversionError, encoded: encoded, rest: rest, kind: :incomplete
+      end
+    rescue _ in ArgumentError ->
```

Comment:
@lexmag Aha. It's actually doced without a list here: http://elixir-lang.org/getting-started/try-catch-and-rescue.html But doced with a list here: http://elixir-lang.org/docs/v1.1/elixir/Kernel.SpecialForms.html#try/1